components/calendar/src/islamic.rs

```diff
@@ -48,14 +48,16 @@ pub struct IslamicCivil;
 #[derive(Copy, Clone, Debug, Default, Hash, Eq, PartialEq, PartialOrd, Ord)]
 #[allow(clippy::exhaustive_structs)]
 /// Umm-al-Qura Hijri Calendar (Used in Saudi Arabia)
-pub struct UmmalQura;
+pub struct UmmAlQura;
 #[derive(Copy, Clone, Debug, Default, Hash, Eq, PartialEq, PartialOrd, Ord)]
 #[allow(clippy::exhaustive_structs)]
 /// A Tabular version of the Arithmetical Islamic Calendar
 pub struct IslamicTabular;
 
 // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/main/calendar.l#L2066
 const FIXED_ISLAMIC_EPOCH_FRIDAY: RataDie = Julian::fixed_from_julian_integers(622, 7, 16);
+
+// Thursday epoch as some communites and calendars consider that Mohammeds migration to Mecca happened the sunset of the 15th.
```

Comment:
I'd have a combined comment before both of these that says something like

"Different islamic calendars use different epochs due to disagreement on the exact date of Mohammed's migration to Mecca."

Reply:
done :)